core-android/src/test/java/com/uber/sdk/android/core/auth/AuthUtilsTest.java

```diff
@@ -47,7 +44,7 @@
     private static final String BEARER = "Bearer";
 
     private final String ACCESS_TOKEN_STRING = "accessToken1234";
-    private final long EXPIRATION_TIME = 1458770906206l;
+    private final long EXPIRATION_TIME = 1458770906206L;
```

Comment:
nit: comment with what this is in human terms?

Reply:
Our linter prefers using an uppercase "L" over lowercase, probably for readability.

Reply:
I meant like, `// august 10th, 48196`